openmc/statepoint.py

```diff
@@ -206,18 +197,15 @@ def _read_criticality(self):
         # Read criticality information
         if self._run_mode == 2:
 
-            self._n_inactive = self._get_int(path='n_inactive')[0]
-            self._gen_per_batch = self._get_int(path='gen_per_batch')[0]
-            self._k_batch = self._get_double(
-                 self._current_batch*self._gen_per_batch,
-                 path='k_generation')
-            self._entropy = self._get_double(
-                 self._current_batch*self._gen_per_batch, path='entropy')
+            self._n_inactive = self._f['n_inactive'].value
+            self._gen_per_batch = self._f['gen_per_batch'].value
+            self._k_batch = self._f['k_generation'].value
```

Comment:
Perhaps the ``_k_batch`` attribute should be changed to ``_k_generation`` to be consistent with the key in the statepoint file?